src/current/v24.3/ldap-authentication.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Configure LDAP Authentication
+summary: Learn how to configure CockroachDB for user authentication using LDAP-compatible directory services such as Active Directory.
+toc: true
+---
+
+CockroachDB supports authentication and authorization using LDAP-compatible directory services, such as Active Directory and Microsoft Entra ID. This allows you to integrate your cluster with your organization's existing identity infrastructure for centralized user management and access control.
+
+This page describes how to configure CockroachDB user authentication using LDAP. You can additionally configure CockroachDB to use the same directory service for user [authorization]({% link v24.3/ldap-authorization.md %}) (role-based access control), assigning CockroachDB roles to users based on their group memberships in the directory.
+
+## Overview
+
+LDAP authentication in CockroachDB uses a "search and bind" approach:
+
+1. CockroachDB first connects to the LDAP server using a service account (bind DN).
+2. It searches for the authenticating user using configurable search criteria.
+3. If found, it attempts to bind to LDAP using the user's credentials.
+4. Upon successful binding, the user is authenticated to CockroachDB.
+
+LDAP authentication in CockroachDB works with any LDAP-compatible directory service, including Microsoft Entra ID, Active Directory, and OpenLDAP. Secure LDAPS connectivity over TLS is supported, and recommended for production environments.
+
+{{site.data.alerts.callout_info}}
+LDAP authentication cannot be used for the `root` user or other admin users. These credentials must be managed separately using password authentication to ensure continuous administrative access, especially if the LDAP service becomes unavailable.
+{{site.data.alerts.end}}
+
+## Configuration
+
+### Prerequisites
+
+1. An LDAP or Active Directory server accessible from your CockroachDB nodes.
+2. A service account (bind DN) with permissions to search the directory.
+3. Network connectivity on port 636 for LDAPS.
+4. The LDAP server's CA certificate if using LDAPS.
```

Comment:
Does this mean you need to download the certificate or get it into your cluster somehow? 

What port will we use if it is not LDAPS?